Sources/Commands/SwiftTestTool.swift

```diff
@@ -280,20 +288,51 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 numJobs: options.numberOfWorkers ?? ProcessInfo.processInfo.activeProcessorCount,
                 diagnostics: diagnostics,
                 options: self.options,
-                buildParameters: try self.buildParameters()
+                buildParameters: buildParameters
             )
             try runner.run(tests)
 
             if !runner.ranSuccessfully {
                 executionStatus = .failure
             }
         }
+
+        if options.shouldEnableCodeCoverage {
```

Comment:
We should factor this code into a method and call it inside each parallel and serial cases. It seems to always run it after this main switch case like this.

Reply:
done